pkg/controller/migmigration/restore.go

```diff
@@ -265,6 +273,10 @@ func (t *Task) hasRestoreCompleted(restore *velero.Restore) (bool, []string) {
 		progress = append(
 			progress,
 			getPodVolumeRestoresProgress(pvrs)...)
+		stageReport, err := t.allStagePodsMatch()
```

Comment:
If this is failed, there is no returning right? I wonder if we maybe don't want to add this check here?

Reply:
Yes, I am a bit skeptical about this too, thought it might help debug the issue if we have this info even if the status is Failed. But I can certainly remove this check.